modules/apps/item-selector/item-selector-test/src/testIntegration/java/com/liferay/item/selector/provider/test/GroupItemSelectorProviderImplTest.java

```diff
@@ -69,6 +70,35 @@ public void testGetGroups() throws Exception {
 		}
 	}
 
+	@Test
+	public void testGetGroupsShouldNotReturnInactiveGroups() throws Exception {
+		List<Group> groups = _groupItemSelectorProvider.getGroups(
+			_group.getCompanyId(), _group.getGroupId(), null, 0, 20);
+
+		int originalGroupsSize = groups.size();
```

Comment:
Instead of assuming there's going to always be <= 20 groups, why not use QueryUtil.POS? Also, shouldn't you update the `getGroupsCount` method as well to return only the number of active groups (the underlying service methods accept a boolean "active" parameter in the map)?

Reply:
if you use the -1 value, the delta is 0 and on https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/portal-kernel/src/com/liferay/portal/kernel/util/ListUtil.java#L165 the server gives an `java.lang.ArithmeticException: / by zero`

That's why on the test are 0, 20